src/modules/marketplaces/marketplaces.elastic.service.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ElasticQuery,
+  QueryType,
+  RangeLowerThan,
+  RangeGreaterThan,
+  ElasticSortOrder,
+  QueryConditionOptions,
+} from '@elrondnetwork/erdnest';
+import { Injectable } from '@nestjs/common';
+import { constants } from 'src/config';
+import { MarketplaceEventsIndexingRequest } from './models/MarketplaceEventsIndexingRequest';
+
+@Injectable()
```

Comment:
why is this a service, you could do something similar to nft-traits.elastic.queries.ts

Reply:
Right, fixed